modules/apps/web-experience/site-navigation/site-navigation-menu-web/src/main/resources/com/liferay/site/navigation/menu/web/portlet/template/dependencies/portlet_display_template_navbar_blank.ftl

```diff
@@ -28,7 +28,11 @@
 
 					<#if showChildren>
 						<#assign nav_item_attr_has_popup = "aria-haspopup='true'" />
-						<#assign nav_item_caret = '<span class="lfr-nav-child-toggle"><i class="icon-caret-down"></i></span>' />
+						<#assign nav_item_caret>
+							<span class="lfr-nav-child-toggle">
+								<@liferay_ui["icon"] icon="angle-down" markupView="lexicon"/>
```

Comment:
@marcoscv-work should this be `<@liferay_aui.icon ...>`?
The `<liferay-ui:icon />` taglib doesn't have a markupView, so I'm wondering if this one worked for you when you tested it?